src/WebJobs.Script/Config/ScriptJobHostOptions.cs

```diff
@@ -108,11 +108,6 @@ public string RootScriptPath
         /// </summary>
         public bool IsSelfHost { get; set; }
 
-        /// <summary>
-        /// Gets or sets retry options to use on function executions on function invocation failures.
-        /// </summary>
-        public RetryOptions Retry { get; set; }
```

Comment:
I see several other things that were added in https://github.com/Azure/azure-functions-host/commit/dd062a5cc9e1f8e8bb875d3e6068b1df3865257a that also need to be removed. Can you check? E.g. ConfigurationSectionNames addition, WellKnownHostJsonProperties.

Reply:
reviewed